open-machine-learning-jupyter-book/deep-learning/nlp/nlp.ipynb

```diff
@@ -46,7 +46,7 @@
     "tags": []
```

Comment:
<p>Why was the associated job deleted? Are there assignments that can be associated, if so please do so.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/739/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
Associated